content/infrastructure/features.md

```diff
@@ -11,36 +11,36 @@ weight = 10
 [[feature]]
   icon = "boxes"
   icon_pack = "fas"
-  name = "An environment for your community"
-  description = "2i2c Hubs provide data environments that have been designed for research and education, and can be customized for your community."
+  name = "Custom environments"
+  description = "Python? R? Julia? Use whatever custom data science stack you want."
```

Comment:
Let's have the patterns similar across items. If we do `thing, thing2, thing3 - ` below, we should do the same pattern here instead of `thing? thing2? thing3`. Or vice-versa

Reply:
Fixed